### PR TITLE
Docs v0.4.52

### PR DESCRIPTION
**Pull request by davidfq**
### Fixes
.

### Features
 - Initial version based on v0.4.52 to support GitBook -> [gitbook PoC for proxy](https://app.asana.com/0/1206952473261039/1206934093284881/f)

### Change implications

 - dependencies added/changed? **no**
 - something important to note in future release notes? **no**

### CODE REVIEW

**Comment by eschultink** [comment]
ok with this ... we should still ultimately squash-merge docs via `rc-` branch. 

i was thinking `docs-` branches or tags to generic specific versions of documentation would follow the naming convention you've used for this branch. eg `docs-v0.4.52` - but should be just a ref to the mainline at that point - ideally a tag, but if that doesn't work, a branch. 

Either case, should just point to the release tag (eg, `v0.4.52`)

File: docs/technical-documentation/README.md
Comment (eschultink) [inline docs/technical-documentation/README.md:1]: calling this `technical-documentation` seems strange to me. does `guides` not make sense?
 
or `how-to`?

File: docs/getting-started.md
Comment (eschultink) [inline docs/getting-started.md:1]: this is really the old index ... not a guide on how to get started with the proxy itself.

File: docs/technical-documentation/api-data-sanitization.md
Comment (eschultink) [inline docs/technical-documentation/api-data-sanitization.md:1]: don't think this really needs to be a symlink; can just put it in some logical sub-directory of `docs`, eg `docs/guides/` or `docs/configuration/` or something.

File: docs/README.md [hunk 1]
```diff
@@ -1,59 +1,326 @@
...
+     your attack surface small and operational complexity low. Furthermore, we define
+     infrastructure-as-code to ease setup.
+  - **transparent** - psoxy's source code is available to customers, to facilitate code review
+     and white box penetration testing.
+  - **simple** - psoxy's functionality will focus on performing secure authentication with the 3rd
+     party API and then perform minimal transformation on the response (pseudonymization, field
+     redaction) to ease code review and auditing of its behavior.
 
-## Guides
+Psoxy may be hosted in [Google Cloud ](docs/gcp/development.md) or [AWS](docs/aws/getting-started.md).
```
Comment (aperez-worklytics) [inline docs/README.md:34]:
```suggestion
Psoxy may be hosted in [Google Cloud](docs/gcp/development.md) or [AWS](docs/aws/getting-started.md).
```

File: docs/README.md [hunk 2]
```diff
@@ -1,59 +1,326 @@
...
-- [Troubleshooting](troubleshooting.md) - some general troubleshooting tips for common issues.
-  - [AWS-specific](aws/troubleshooting.md)
-  - [GCP-specific](gcp/troubleshooting.md)
-- [Cleaning Up](guides/cleaning-up.md) - use Terraform to clean up (destroy) your deployment when you're
-  done with it.
+## Data Flow
 
-### Data Sources
+A Psoxy instances reside on your premises (in the cloud) and act as an intermediary between
+Worklytics and the data source you wish to connect.  In this role, the proxy performs the
```
Comment (aperez-worklytics) [inline docs/README.md:39]:
```suggestion
Worklytics and the data source you wish to connect. In this role, the proxy performs the
```

File: docs/README.md [hunk 3]
```diff
@@ -1,59 +1,326 @@
...
+    "id": "t~A80SJXrbfawKpDRcddGnKI4QDKyjQI9KtjJZDb8FZ27UE_toS68FyWz7Y22fnQYLP91SHJ",
+    "email": "p~SIoJOpeSgYF7YUPQ28IWZexVuHyN9A80SJXrbfawKpDRcddGnKI4QDKyjQI9KtjJZDb8FZ27UE_toS68FyWz7Y22fnQYLP91SHJGVwQiN3E@acme.com"
+}
+```
+These pseudonyms leverage SHA-256 hashing / AES encryption, with salt/keys that are known only to
+your organization and never transferred to Worklytics.
 
-- [Deployment Migration](guides/deployment-migration.md)
+Psoxy enforces that Worklytics can only access API endpoints you've configured (principle of least
+privilege) using HTTP methods you allow (eg, limit to `GET` to enforce read-only for RESTful APIs).
```
Comment (aperez-worklytics) [inline docs/README.md:68]: Worth to include a wikipedia link with the defintiion of PLP? 

```suggestion
Psoxy enforces that Worklytics can only access API endpoints you've configured ([principle of least
privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege)) using HTTP methods you allow (eg, limit to `GET` to enforce read-only for RESTful APIs).
```

**Review by aperez-worklytics — APPROVED** [review]

**Pull request merged**